portfolio/src/main/webapp/style.css

```diff
@@ -1,14 +1,171 @@
+.active,
+.demo:hover {
+  opacity: 1;
+}
+
+.avatar {
+  border-radius: 50%;
+  margin: 5px;
+  position: absolute;
+  right: 8px;
+  top: 4px;
+  vertical-align: right;
+  width: 100px;
+}
+
+body {
+  background-color: #555;
+}
+
+body, html {
+  font-family: Arial, Helvetica, sans-serif;
+  height: 100%;
+  margin: 0;
+}
+
+/* Container for image text */
+.caption-container {
+  background-color: #222;
+  color: white;
+  padding: 2px 16px;
+  text-align: center;
+}
+
+/* Six columns side by side */
+.column {
+  float: left;
+  width: 16.66%;
+}
+
 #content {
   margin-left: auto;
   margin-right: auto;
   width: 650px;
 }
 
+/* Add a pointer when hovering over the thumbnail images */
+.cursor {
+  cursor: pointer;
+}
+
+/* Add a transparency effect for thumnbail images */
+.demo {
+  opacity: 0.6;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
 
+.hero-image-container {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), 
+                    url("images/friends/friend-1.JPG");
+  background-position: center;
+  height: 50%;
+  margin-top: 5px;
+  position: relative;
+}
+
+.hero-text {
+  color: white;
+  text-align: center;
+  position: absolute;
+  width: 100%;
+  margin-top: 200px;
+}
+
 #map {
   height: 378px;
   width: 504px;
 }
+
+/* Hide the images by default */
+.mySlides {
+  display: none;
+}
+
+/* Style the navbar */
+#navbar {
+  overflow: hidden;
+  background-color: #333;
+}
+
+/* Navbar links */
+#navbar a {
+  color: #f2f2f2;
+  display: block;
+  float: left;
+  padding: 14px;
+  text-align: center;
+  text-decoration: none;
+}
+
+/* Next & previous buttons */
+.prev,
+.next {
+  border-radius: 0 3px 3px 0;
+  color: white;
+  cursor: pointer;
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: -50px;
+  padding: 16px;
+  position: absolute;
+  top: 40%;
+  user-select: none;
+  width: auto;
+  -webkit-user-select: none;
+}
+
+/* On hover, add a black background color with a little bit see-through */
+.prev:hover,
+.next:hover {
+  background-color: rgba(0, 0, 0, 0.8);
+}
+
+/* Position the "next button" to the right */
+.next {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* Number text (1/3 etc) */
+.numbertext {
+  color: #f2f2f2;
+  font-size: 12px;
+  padding: 8px 12px;
+  position: absolute;
+  top: 0;
+}
+
+.row:after {
+  clear: both;
+  content: "";
+  display: table;
+}
+
+/* Position the image container (needed to position the left and right arrows) */
+.slideshow-container {
+  position: relative;
+}
+
+/* The sticky class is added to the navbar with JS when it reaches its 
```

Comment:
nit: The first line of multiline comments should just be `/*`, the text starts on the next line. https://google.github.io/styleguide/jsguide.html#formatting-block-comment-style